source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -665,8 +665,8 @@ The event API here is assumed to be like the standard `Python Event
             # was successful and included the topologyVersion field, or the
             # previous response included the moreToCome flag, or the server
             # has just transitioned to Unknown from a network error.
-            if (description.type != Unknown and description.topologyVersion != Null or
-                    connection.moreToCome or
+            if (description.type != Unknown and description.topologyVersion != Null) or
+                    (conn is not None && connection.moreToCome) or
```

Comment:
We already use Null instead of None for topologyVersion, can this be `connection != Null`? And can you update line 645 to be `connection = Null`?

Reply:
Done.